internal/policies/mount/entries_test.go

```diff
@@ -10,13 +10,36 @@ import (
 var EntriesForTests = map[string]entry.Entry{
 	"entry with one value": {Value: "protocol://domain.com/mountpath"},
 
+	"entry with one good value and one bad": {Value: `
+protocol://domain.com/mountpath
+protocol//bad.com/badpath
+`,
+	},
+
+	"entry with one anonymous value": {Value: "[anonymous]protocol://anon.com/anon/mount"},
+
 	"entry with multiple values": {Value: `
 protocol://domain.com/mountpath2
 smb://otherdomain.com/mount/path
 nfs://yetanotherdomain.com/mount-path/mount/path
 `,
 	},
 
+	"entry with multiple slightly different values": {Value: `
+smb://otherdomain.com/mount/path
```

Comment:
Can we add something like "This is another value" so that it’s crystal clear when reading the golden files the entries were updated? (it’s annoying to have to check for the definition for every single golden files), maybe you can take the same approach for some other use case (as you did for bad.com, which is clear it’s a bad line).

Reply:
I thought about it, but I specifically decided to use this values to see the escape function in action. If you look closely, you'll see that the only changes are in slashes and dashes positions...

Reply:
but it makes hard for a refresh to see it, maybe some are on the unit test level to check the escaping part and for package tests, we have something more descriptive?

Reply:
Yes. Didier wanted me to simplify the tests a little bit, since they had too many different inputs with no real gain.